src/datatypes/src/vectors/null.rs

```diff
@@ -196,6 +196,10 @@ impl MutableVector for NullVectorBuilder {
         self.length += length;
         Ok(())
     }
+
+    fn push_null(&mut self) {
+        self.push_value_ref(ValueRef::Null);
```

Comment:
```suggestion
        self.length += 1;
```

Reply:
ohh.. that's cool! should have spent a bit more time looking at the other methods within `NullVectorBuilder`